excalidraw-app/scripts/woff2-vite-plugins.js

```diff
@@ -5,7 +5,7 @@ const OSS_FONTS_CDN =
  * Custom vite plugin to convert url woff2 imports into a text.
  * Other woff2 imports are automatically served and resolved as a file uri.
  *
- * @returns {import("vite").PluginOption}
+ * @returns {import("vitest").PluginOption}
```

Comment:
Is this change necessary? In reality, it's a vite plugin, not just vitest, as it's executed during build.

Reply:
No, it's just a type clash. I can change it back.